api/cloud_accounts_aws_agentless_scanning.go

```diff
@@ -0,0 +1,57 @@
+//
+// Author:: Darren Murray(<darren.murray@lacework.net>)
+// Copyright:: Copyright 2022, Lacework Inc.
+// License:: Apache License, Version 2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package api
+
+// GetAwsAgentlessScanning gets a single AwsAgentlessScanning integration matching the provided integration guid
+func (svc *CloudAccountsService) GetAwsAgentlessScanning(guid string) (
```

Comment:
I'm sorry but we need to follow naming convention for the CloudAccount type and function(s)
```suggestion
func (svc *CloudAccountsService) GetAwsAwsSidekick(guid string) (
```